andela/ah-rambo-frontend
# #167190493 User View Profile

#### What does this PR do?
This PR enables a registered user to view their profile
#### Description of Task proposed in this pull request?
- Profile Reducers
- Profile actions
- UserProfile container components
- sideNav component
- userCard component
- userArticle component
- DashBoard Navigation component

#### How should this be manually tested (Quality Assurance)?
```
localhost:8080/profile
```

#### What are the relevant pivotal tracker stories?
- [167190493](https://www.pivotaltracker.com/story/show/167190493)

#### Any background context you want to add (Operations Impact)?
To test the following is required at first
```
npm i
npm run start:dev
```
- Reusable components such as sideNav, DashBoard Navigation, userdetails Card have been implemented in this PR
- To test ensure there's stable network, API calls were made.

#### What I have learned working on this feature:
I learnt about redux and react

#### Screenshots: 
<img width="1440" alt="Screenshot 2019-09-05 at 6 06 15 AM" src="https://user-images.githubusercontent.com/44194416/64313395-69040200-cfa3-11e9-87da-3fa786d9866b.png">
